packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -511,7 +510,7 @@ $Calendar-dayPicker-margin: 10px;
 
 .wrap span:focus,
 .wrap div:focus {
-  outline: 1px solid $ms-color-themePrimary;
+  outline: $ms-color-themePrimary;
```

Comment:
It does not add a border to the element itself, it uses a pseudo border, creating an :after element with a border on it.